lms/static/js/learner_dashboard/views/certificate_view.js

```diff
@@ -0,0 +1,33 @@
+;(function (define) {
+    'use strict';
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'text!../../../templates/learner_dashboard/certificate.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             certificateTpl
+         ) {
+            return Backbone.View.extend({
+                el: '.certificates-list',
+                tpl: _.template(certificateTpl),
+                initialize: function(data) {
+                    this.context = data.context;
+                    this.certificatesData = this.context.certificatesData;
+                    this.xseriesImage = this.context.xseriesImage;
```

Comment:
These 2 variables are not needed, this.context can be passed directly into the template.